src/data/cond.ml

```diff
@@ -64,9 +64,9 @@ module Operation = struct
   let false_ = Value Objt.false_
   let and_ x y = op2 x Op.And_ y
   let and_s x y =
-    if x = Value (Objt.true_) then
+    if Stdlib.(x = Value) (Objt.true_) then
       y
-    else if y = Value (Objt.true_) then
+    else if Stdlib.(y = Value) (Objt.true_) then
```

Comment:
Same

```diff
- else if Stdlib.(y = Value) (Objt.true_) then
+ else if Stdlib.(y = Value Objt.true_) then
```

Reply:
Oops, I pushed the wrong commit.
Your fix seems to be correct, and I have pushed it to my repository.

I think the failure is caused by some dependency problem.
Could you try to recompile it after the command "omake clean", or from a re-cloned directory?